### DISCUSSION TITLE
组件（插件）版本锁

### DISCUSSION THREAD

**Discussion by FoundTheWOUT** [the1812/Bilibili-Evolved#2891] [功能建议 / Ideas]
开发时发现的问题，估摸着线上版也会发生
某插件依赖新版`core`的`api`，但`core`未更新，组件就更新了，会导致一些较难发现的错误。
是不是让组件（插件）的版本小于等于当前`core`的版本更好。

**Comment by the1812** [comment]
更新这块现在很不完善... core 的更新提示都还没做(
